rest/configs/PaypalSandbox.oauth

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Settings>
+  <AuthorizationRequestUrl>https://www.sandbox.paypal.com/webapps/auth/protocol/openidconnect/v1/authorize?client_id={ApiKey}&amp;response_type=code&amp;scope=profile+email+address+phone+https://uri.paypal.com/services/paypalattributes&amp;redirect_uri=http://pragmaticworks.com</AuthorizationRequestUrl>

```

Comment:
This uses a pragmaticworks.com address for the redirect_uri. That should probably be localhost. Please check the other files to make sure we don't have any other uses.

Reply:
Thank you for pointing that out, I've updated them all to the new one